### PR TITLE
Updated AuthorizationPolicies

### PR DESCRIPTION
**Pull request by Ambertvu**
## Description

I've clarified the usage of AuthorizationPolicies and added a bit on how to apply it.

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [x] New content
* [ ] Updates related to a new version
* [ ] Other

### CODE REVIEW

File: 11/umbraco-cms/reference/routing/umbraco-api-controllers/authorization.md [hunk 1]
```diff
@@ -40,6 +40,49 @@ public class ProductsController : UmbracoApiController
 }
 ```
 
+The `AuthorizationPolicies` has [quite a few](https://github.com/umbraco/Umbraco-CMS/blob/contrib/src/Umbraco.Web.Common/Authorization/AuthorizationPolicies.cs) other options you can set. A example is `UserBelongsToUserGroupInRequest`. Using this policy you can check if the current incoming request of the user is in a specific backoffice usergroup.
```
Comment (sofietoft) [inline 11/umbraco-cms/reference/routing/umbraco-api-controllers/authorization.md:43]: A couple of minor adjustments to clean up the vale warning. 

```suggestion
The `AuthorizationPolicies` has a [series of other options](https://github.com/umbraco/Umbraco-CMS/blob/contrib/src/Umbraco.Web.Common/Authorization/AuthorizationPolicies.cs) you can use. An example is `UserBelongsToUserGroupInRequest`. Using this policy you can check if the current incoming request of the user is in a specific backoffice User Group.
```

**Review by sofietoft — COMMENTED** [review]
Thanks for the PR @Ambertvu ! 💪 
I've made a small suggestion - let me know what you think

**Comment by Ambertvu** [comment]
All good @sofietoft, thanks!

**Comment by sofietoft** [comment]
Thanks for considering the suggestion @Ambertvu ! 💪 

Looks like everything checks out.
I'll run it by our developers for a quick check as well!

**Comment by elit0451** [comment]
I had to make changes to your updated articles @Ambertvu  - https://github.com/umbraco/UmbracoDocs/commit/30ced67458de7200b76d76a60f7d475bf13cf4cc

I don't have permissions to push directly to your fork, so if you cherry-pick the commit above we can merge this PR 🙂

**Comment by Ambertvu** [comment]
I hope I've done this correctly :-)

File: 11/umbraco-cms/reference/routing/umbraco-api-controllers/authorization.md [hunk 2]
```diff
@@ -40,6 +40,52 @@ public class ProductsController : UmbracoApiController
 }
 ```
 
+The `AuthorizationPolicies` has a [series of other options](https://apidocs.umbraco.com/v11/csharp/api/Umbraco.Cms.Web.Common.Authorization.AuthorizationPolicies.html) other options you can set. An example is `UserBelongsToUserGroupInRequest`. By using this policy, you can check if the current incoming request of the user is in a specific backoffice User Group.
```
Comment (elit0451) [inline 11/umbraco-cms/reference/routing/umbraco-api-controllers/authorization.md:43]:
```suggestion
The `AuthorizationPolicies` has a [series of other options](https://apidocs.umbraco.com/v11/csharp/api/Umbraco.Cms.Web.Common.Authorization.AuthorizationPolicies.html) you can set. An example is `UserBelongsToUserGroupInRequest`. By using this policy, you can check if the current incoming request of the user is in a specific backoffice User Group.
```

**Review by elit0451 — COMMENTED** [review]
There is just a single thing we need to fix (_I've made a suggestion about it below_) and then this article is good to go 💪 

Great work @Ambertvu 🙂

**Comment by Ambertvu** [comment]
Nice find! Hah, thanks!

**Comment by sofietoft** [comment]
Thanks for applying the final suggestion here @Ambertvu ! 💪 
I can see Elitsa is off on (well-deserved!) vacation for the next few weeks, so I'll step in here.

Can see that we're ready for merging - so I'll do that.
Thanks again for taking the time to make the PR 🥳 MUCH appreciated!

Just 1 more thing:
This PR is adding to the v11 docs - should it be added to 12 as well?
If yes, I'll copy it over later today. Let me know 💪

**Comment by Ambertvu** [comment]
I'm so slacky, sorry, buzy with 101 things, hah!

But yes, definitly also applicable for V12!

**Comment by sofietoft** [comment]
No worries @Ambertvu ! 😄 
We're all busy, busy people 😁 

I'll make sure this is also added to the latest version of the docs 💪

**Pull request merged**